Barotrauma/BarotraumaShared/SharedSource/Items/Components/Signal/GreaterComponent.cs

```diff
@@ -33,15 +36,18 @@ public override void Update(float deltaTime, Camera cam)
 
         public override void ReceiveSignal(Signal signal, Connection connection)
         {           
-            //base.ReceiveSignal(signal, connection);
             switch (connection.Name)
             {
                 case "signal_in1":
-                    float.TryParse(signal.value, NumberStyles.Float, CultureInfo.InvariantCulture, out val1);
+                    float signal1 = float.NegativeInfinity;
+                    float.TryParse(signal.value, NumberStyles.Float, CultureInfo.InvariantCulture, out signal1);
+                    val1 = signal1 > val1 ? signal1 : val1;
                     timeSinceReceived[0] = 0.0f;
                     break;
                 case "signal_in2":
-                    float.TryParse(signal.value, NumberStyles.Float, CultureInfo.InvariantCulture, out val2);
+                    float signal2 = float.PositiveInfinity;
+                    float.TryParse(signal.value, NumberStyles.Float, CultureInfo.InvariantCulture, out signal2);
+                    val2 = signal2 < val2 ? signal2 : val2;
```

Comment:
I believe this breaks backwards compatibility too. Previously the last received value was used, now the min/max value is used.